libsel4simple/arch_include/arm/simple/arch/simple.h

```diff
@@ -16,7 +16,13 @@
 
 /* Simple does not address initial null caps, including seL4_CapNull.
  * seL4_CapIOSpace, seL4_CapIOPortControl are null on architectures other than x86 */
-#define SIMPLE_SKIPPED_INIT_CAPS 3
+#ifdef CONFIG_KERNEL_MCS
+#define SKIP_THREADSC 0
```

Comment:
Maybe better prefix all the new defined with `SIMPLE_` to have a kind of namespace.

Reply:
Good idea. I will do that.

Reply:
Done